src/Momento.Sdk/ISimpleCacheClient.cs

```diff
@@ -86,7 +86,7 @@ public interface ISimpleCacheClient : IDisposable
     /// <param name="cacheName">Name of the cache to store the item in.</param>
     /// <param name="key">The key to set.</param>
     /// <param name="value">The value to be stored.</param>
-    /// <param name="ttlSeconds">TTL for the item in cache. This TTL takes precedence over the TTL used when initializing a cache client. Defaults to client TTL.</param>
+    /// <param name="ttlSeconds">TTL for the item in cache. This TTL takes precedence over the TTL used when initializing a cache client. Defaults to client TTL. If specified must be strictly positive.</param>
```

Comment:
why should we regress the C# experience now instead of adding a compatibility layer for less-featured languages later?

Reply:
Leaving language compatibility aside, I see this as in improvement in the experience.

While I agree the domain of values of `uint` is better suited for this than the domain of `int`, we've received feedback from two independent users that expressing TTL as an `int` is a pain because they would like to use native, ergonomic time span objects to express the TTL, eg:

```csharp
client.SetAsync("my cache", "my key", "my value", new TimeSpan(hours: 24, minutes: 0, seconds: 0).TotalSeconds);
```

but those `TimeSpan` objects deal in `int`, not `uint`, so they would have to add [checked casts](https://learn.microsoft.com/en-us/dotnet/csharp/language-reference/statements/checked-and-unchecked) everywhere, or else the compiler complains. In reality these casts are a don't care, because the range of values `uint` contains that `int` does not widely exceeds our max TTL.

We've received feedback from an expert user that `uint` is odd to see in .NET APIs.

Reply:
Huh, til my c# experience is irredeemably out of date with the style of the cool kids. Do we at least still think Jon Skeet is the bee's knees?

Maybe instead of a domain type _regression_ the _addition_ of a TimeSpan ttl option would be better? For example I would rather write
```
client.SetAsync("my cache", "my key", "my value", new TimeSpan(hours: 24, minutes: 0, seconds: 0));
```

than
```
client.SetAsync("my cache", "my key", "my value", new TimeSpan(hours: 24, minutes: 0, seconds: 0).TotalSeconds);
```

and tbh, TimeSpan might be the overwhelmingly preferred TTL type option - though that suspicion is based on an admittedly stale experience.

Reply:
also would let me write:
```
client.SetAsync("my cache", "my key", "my value", currentCacheRefreshMark - DateTime.Now);
```

to coordinate item lifetimes across my cache.